packages/lesswrong/components/review/FrontpageReviewWidget.tsx

```diff
@@ -140,51 +140,57 @@ const FrontpageReviewPhase = ({classes}: {classes: ClassesType}) => {
   const currentDate = moment(new Date())
   const activeRange = getReviewPhase()
 
+  const forumTitle = forumTitleSetting.get()
+
+  const nominationPhaseDateRange = <span>{nominationStartDate.format('MMM Do')} – {nominationEndDate.format('MMM Do')}</span>
+  const reviewPhaseDateRange = <span>{nominationEndDate.clone().add(1, 'day').format('MMM Do')} – {reviewEndDate.format('MMM Do')}</span>
+  const votingPhaseDateRange = <span>{reviewEndDate.clone().add(1, 'day').format('MMM Do')} – {voteEndDate.format('MMM Do')}</span>
+
   // EA will use LW text next year, so I've kept the forumType genericization
   const overviewToolip = isEAForum ?
-    <>
+    <div>
```

Comment:
Huh, I'm curious what the issue was?

Reply:
I think I just twiddled that randomly while trying to fix something, which it didn't help with, but then didn't feel motivated to change it back.